File: src/main/java/fr/kabiro/lol/ism/core/model/match/MatchSummary.java
package fr.kabiro.lol.ism.core.model.match;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder
public class MatchSummary {
    private Long gameId;
    private String gameMode;
    private String platformId;
    private Long gameStartTimestamp;
    private Team teamA;
    private Team teamB;
}

File: src/main/ui/app/scripts/model/MatchSummary.ts
export interface MatchSummary {
    gameId: string ;
    gameMode: string ;
    platformId: string ;
    gameStartTimestamp: number ;
    teamA: Team ;
    teamB: Team ;
}

export interface Team {
    players: TeamPlayer[];
}

export interface TeamPlayer {
    championId: number;
    championName: string;
}
